pod_identity.tf

```diff
@@ -0,0 +1,70 @@
+locals {
```

Comment:
I would prefix all local variables in this file with `pod_identity_` to prevent collisions.

Reply:
Done.